### ISSUE TITLE
Cant deny permissions

### ISSUE THREAD

**Issue by MPeti1** [RikkaApps/App-Ops-issue-tracker#13]
Tried with messenger use camera permission. Restarted messenger process after setting the permission to deny. I use the magisk module mode

**Comment by MPeti1** [comment]
+info:
System: MIUI 8 7.6.15
Root: magisk 16.0
Xposed: solarwarez edition
App Ops: 2.3.7.r576

**Comment by MPeti1** [comment]
+info:
System: MIUI 8 7.6.15
Root: magisk 16.0
Xposed: solarwarez edition
App Ops: 2.3.7.r576

**Comment by timea-techgirl** [comment]
I have the same problem.
All settings are allowed by MIUI even if I deny it.

**Comment by MPeti1** [comment]
It seems working for me now, Even with messenger.

Although now as I went through messenger's permissions again I noticed that I forgot to set it up after the reinstall, and that messenger have **sent an SMS** without my knowledge according to AppOps.

@RikkaW how precise are the last used statistics usually? Is it possible that allowing WRITE_SMS permission doesn't necessarily mean an SMS has been sent?

![image](https://user-images.githubusercontent.com/9215491/66266958-df5a8500-e82b-11e9-9937-e7f6472adb77.png)
